### ISSUE TITLE
Cannot add certain conditions that are nested inside of "state" for Lutron Aurora Dimmer

### ISSUE THREAD

**Issue by agentorange96** [Hyrules/WinHue#278]
### Your setup 
WinHue version : 3.0.4773.0
Bridge api version : 1.35
Sensor model : Lutron Aurora

### Expected behavior
In the visual rule editor, I select my sensor, state->value->expectedrotation
I do something with this. (equal, gt/lt/etc) I click add and it applies it to state->value->expectedrotation

### Actual behavior
In the visual rule editor, I select my sensor, state->value->expectedrotation
I do something with this. (equal, gt/lt/etc) I click add and it applies it to state
This doesn't work because state doesn't accept this as a value.

### Steps to reproduce the problem
1) Create a new rule
2) Select a sensor: A Lutron Aurora dimmer
3) Attempt to set a rule based on something inside of state->value
    For example state->value->expectedrotation
4) Click "Add"

**Comment by Hyrules** [comment]
Can you post a json export of this particular sensor ? I would like to see the structure of this sensor. thanks.

**Comment by agentorange96** [comment]
Yeah, no problem! I am able to modify existing rules with this switch without WinHue complaining if that's helpful information. (So I just made a rule in another program and modified it with the editor, since I was too lazy to learn the advanced editor.)
This particular switch actually uses two sensors, so I made sure to include both parts. However, the button sensor is pretty standard and WinHue handles it just fine.
Thank you!
[LutronAurora.txt](https://github.com/Hyrules/WinHue/files/4181153/LutronAurora.txt)

**Comment by Hyrules** [comment]
Ok first time I see this kind of sensor. Problem with sensors is that they seem to have different properties from sensor to sensor. It's normal that you can see the sensor and it's kinda normal that you cannot select is properties because they don't exist in WinHue. I will have to find a way to deal with this in WinHue and not having to create a new sensor everytime I stumble upon someone with a new sensor kind. It's borderline bug and feature request. I'm not actively developping WinHue 3 at the moment just trying to fix bugs as they come. I'm working ( working is an understatement at the moment) on a new version of WinHue with .net core 3.X trying to improve on what's already done. I have an idea how to deal with this. I will see what I can do about this issue.

**Comment by agentorange96** [comment]
Yeah, at the very least it might be good to have dynamic sensors so to speak in V4. (Even if you don't get to it for V3) I don't think it's reasonable to expect one person working on this, I'd assume in your free time, to be able to keep up with every sensor that comes out. I very much appreciate your work btw! 😀

**Comment by Hyrules** [comment]
Thanks. I think my idea will work. for the next version. A word of advice though if you use an old bridge I suggest you get a new one because WinHue will not support them as soon as I have the time to release the next version for the WH3 i mean.
